### ISSUE TITLE
Dropbox installer sets pkg metadata to date of January 1, 1980 on install

### ISSUE THREAD

**Issue by smashism** [autopkg/MLBZ521-recipes#74]
Hey there! Thank you for creating a meta installer for both architectures for Dropbox!!!

I'm testing deployments of the pkg that is created with this recipe and noticed that the App shows a last modified date of Jan 1, 1980. Digging into the pkg I also see that date on the `Info.plist` and `PkgInfo` are also Jan 1, 1980. 

Based some digging this _appears_ to be a Dropbox issue and not necessarily an autopkg/recipe issue, but I wanted to raise it here so you and others might be aware and hopefully Dropbox fixes it?

Timestamps from the app after installing the app on a Mac (running 15.1 fwiw):
![image](https://github.com/user-attachments/assets/2b59e999-db7d-414e-acdf-09858c70fb8c)

Checking `mdls` on the app bundle downloaded directly from Dropbox:
![image](https://github.com/user-attachments/assets/f6f246af-fba4-4705-9b98-907edc36425b)

**Comment by MLBZ521** [comment]
Yeah, I don't know any reason why this would be caused by anything in my recipes, but thanks for the FYI.
